src/server.h

```diff
@@ -1098,13 +1098,15 @@ struct clusterState;
 #undef hz
 #endif
 
-#define CHILD_INFO_MAGIC 0xC17DDA7A12345678LL
 #define CHILD_TYPE_NONE 0
 #define CHILD_TYPE_RDB 1
 #define CHILD_TYPE_AOF 2
 #define CHILD_TYPE_LDB 3
 #define CHILD_TYPE_MODULE 4
 
+#define COW_TYPE_ACTIVE 0
```

Comment:
let's rename this too.
i.e. "active" means it's currently active, but maybe "current" complements "last" better.
```suggestion
#define COW_TYPE_CURRENT 0
```